src/Compilers/Core/CodeAnalysisTest/FileUtilitiesTests.cs

```diff
@@ -268,5 +268,48 @@ public void Extension()
 
             Assert.Equal("*", PathUtilities.RemoveExtension("*.dll"));
         }
+
+        [ConditionalTheory(typeof(WindowsOnly))]
+        [InlineData("x:\\a\\b\\file.cs", "x:/a/b/file.cs")]
```

Comment:
>\\ [](http://example.com/codeflow?start=23&length=2)

Consider using verbatim strings for each of these values, to avoid the need to escape backslashes so it's more obvious what we're testing in each case.